Makefile

```diff
@@ -462,4 +473,7 @@ endif
 ifdef CONSISTENCY_LEVEL
 	sed -i -e 's/cl=one/cl=$(CONSISTENCY_LEVEL)/g' /tmp/cassandra-stress-$(STRESS_TYPE).yaml
 endif
+
+	cat /tmp/cassandra-stress-$(STRESS_TYPE).yaml
```

Comment:
Remove debug statement

Reply:
I found it useful to see the real conf that is going to be applied when launching the command